frontend/src/initKea.ts

```diff
@@ -79,7 +79,7 @@ export function initKea({ routerHistory, routerLocation, beforePlugins }: InitKe
                 if (
                     !ERROR_FILTER_WHITELIST.includes(actionKey) &&
                     (error?.message === 'Failed to fetch' || // Likely CORS headers errors (i.e. request failing without reaching Django)
-                        (error?.status !== undefined && ![200, 201, 204].includes(error.status)))
+                        (error?.status !== undefined && ![200, 201, 204, 409].includes(error.status)))
```

Comment:
We seem to show the toast on every request. 409 doesn't seem to be used anywhere else and should be handled gracefully so thought it worth adding as a new special case here

Reply:
Hmmm I don't know that this wouldn't cause issues elsewhere...

A more localized way would be to simply catch the error from the API call in the logic and check for the error status.

Reply:
That would mean the little toast appears in the bottom right hand corner which seemed uncessary given the error is handled in the UI
![Screenshot 2023-07-03 at 15 09 03](https://github.com/PostHog/posthog/assets/6685876/52726b8b-dc82-4540-a359-c8623bbf2e08)

The other option would be to add the `saveNotebook` action to the `ERROR_FILTER_WHITELIST` array. That has the effect of not showing the toast for any failed response from the server for that action which felt less good e.g. an actual 500 would not show anything in the UI despite having failed to save
https://github.com/PostHog/posthog/blob/0bb4898d3a946ea79c9b13375b2d33d3f54ee752/frontend/src/initKea.ts#L12-L16

What we really want is a combination of the `saveNotebook` and `409 response`. It doesn't look like there is any way to inject conditions given it is a global listener 🤔

Reply:
Actually...

Maybe I can implement something within the logic. It seems to get passed as an argument to the `onFailure` callback

Reply:
Please ignore... covered in https://github.com/PostHog/posthog/pull/16352#discussion_r1250887110